script.skin.helper.service/default.py

```diff
@@ -99,36 +114,50 @@ def __init__(self):
                 if resultAction:
                     if "jsonrpc" in resultAction:
                         xbmc.executeJSONRPC(resultAction)
+                        xbmc.executeJSONRPC(resultAction)
```

Comment:
Well, this may sound silly but I have discovered that the call fails from time to time.
Been playing with it for hours and this was the only "solution".

This will launch a movie/episode from the library if the play button is clicked in the special infodialog. It may be related to the fact that a play action is launched when the dialog is still closing but I couldn't find any documentation about that